### PR TITLE
add new dapp list

### PR DESCRIPTION
**Pull request by andrewmarushko**
https://www.notion.so/madfissolutions/Development-05abe2b65cd14ae39a101533bc01b708?p=2984d44f213d4bc4b79a75903ff3627b

### CODE REVIEW

File: src/api.service.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ export const bakingBadApi = axios.create({ baseURL: 'https://api.baking-bad.org/
 
 export const tzktApi = axios.create({ baseURL: 'https://api.mainnet.tzkt.io/v1' });
 
-export const templeWalletApi = axios.create({ baseURL: 'https://api.templewallet.com/api' });
+// export const templeWalletApi = axios.create({ baseURL: 'https://api.templewallet.com/api' });
+export const templeWalletApi = axios.create({ baseURL: 'http://localhost:3000/api' });
```
Comment (unixvb) [inline src/api.service.ts:10]: fix this

File: src/interfaces/custom-dapps-info.interface.ts [hunk 2]
```diff
@@ -1,28 +1,12 @@
...
-  full_description: string;
-  website: string;
-  slug: string;
-  authors: string[];
-  social_links: string[] | null;
-  interfaces: string[] | null;
-  categories: string[];
-  soon: boolean;
+  dappUrl: string;
+  type: string;
```
Comment (unixvb) [inline src/interfaces/custom-dapps-info.interface.ts:4]: use enum

File: src/interfaces/custom-dapps-info.interface.ts [hunk 3]
```diff
@@ -1,28 +1,12 @@
...
   logo: string;
-  cover: string;
-  screenshots?: BcdDAppScreenshot[];
-  tvl: string;
-  errorOccurred?: boolean;
+  slug: string;
+  category: string[];
```
Comment (unixvb) [inline src/interfaces/custom-dapps-info.interface.ts:7]: use enum

File: src/store/d-apps/d-apps-epics.ts [hunk 4]
```diff
@@ -100,3 +100,4 @@ const loadDAppsListEpic = (action$: Observable<Action>) =>
   );
 
 export const dAppsEpics = combineEpics(loadPermissionsEpic, removePermissionEpic, abortRequestEpic, loadDAppsListEpic);
+//
```
Comment (unixvb) [inline src/store/d-apps/d-apps-epics.ts:103]: remove this

**Review by letier3110 — APPROVED** [review]

File: src/api.service.ts (hunk 1 above)
Comment (andrewmarushko) [reply src/api.service.ts]: Done!

File: src/interfaces/custom-dapps-info.interface.ts (hunk 2 above)
Comment (andrewmarushko) [reply src/interfaces/custom-dapps-info.interface.ts]: done

File: src/store/d-apps/d-apps-epics.ts (hunk 4 above)
Comment (andrewmarushko) [reply src/store/d-apps/d-apps-epics.ts]: Done

File: src/interfaces/custom-dapps-info.interface.ts [hunk 5]
```diff
@@ -1,28 +1,20 @@
...
-  soon: boolean;
+  dappUrl: string;
+  type: DappType;
   logo: string;
-  cover: string;
-  screenshots?: BcdDAppScreenshot[];
-  tvl: string;
-  errorOccurred?: boolean;
+  slug: string;
+  category: [DappType];
```
Comment (unixvb) [inline src/interfaces/custom-dapps-info.interface.ts:15]: `DappType[]` and `[DappType]` is a different thing

**Review by unixvb — APPROVED** [review]

File: src/interfaces/custom-dapps-info.interface.ts [hunk 6]
```diff
@@ -1,28 +1,23 @@
...
-  soon: boolean;
+  dappUrl: string;
+  type: DappType;
   logo: string;
-  cover: string;
-  screenshots?: BcdDAppScreenshot[];
-  tvl: string;
-  errorOccurred?: boolean;
+  slug: string;
+  category: DappType[];
```
Comment (unixvb) [inline src/interfaces/custom-dapps-info.interface.ts:18]: just use `category: DappEnum[];` instead of `DappType`

**Review by unixvb — APPROVED** [review]

**Pull request merged**